lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,9 +8,12 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
           attr_accessor :#{column}_data_filename, :#{column}_data_mimetype
 
           def #{column}_data_uri=(data)
+            return if data.blank?
```

Comment:
@timsly thanks for your review.

But, I have no idea to accomplish case of blank data with form.
I don't want to parse `data` when data is blank.
If it raise an error, I can't update model by form.

I rethink more good solution.
thanks a lot.